packages/components/src/components/select/single-select/select.scss

```diff
@@ -79,6 +79,17 @@
       -webkit-user-select: none;
       -ms-user-select: none;
       user-select: none;
+
+      div {
```

Comment:
Which `div` are you styling? Please add a class, and don't apply styles to all `<div>`s.

Reply:
The problem regarding the style was this in the react example (App.scss):
```scss
div {
  background-color: tokens.$ifxColorBaseWhite;
  // --ifx-font-family: 'Arial black'; //overwrite default font
}
```

I wasn't sure whether I should remove it in the example, or override it in the component, but I'll change it.